projects/observability/src/public-api.ts

```diff
@@ -82,7 +82,7 @@ export * from './pages/apis/api-detail/traces/api-trace-list.component';
 export * from './pages/apis/api-detail/traces/api-trace-list.dashboard';
 export * from './pages/api-trace-detail/api-trace-detail.page.component';
 
-export * from './pages/apis/services/service-list.module';
+export * from './pages/apis/endpoints-list.module';
```

Comment:
I'm shocked you didn't put this with the component and dashboard in the same dir 😉

Reply:
Hah, fail. It was actually because it started out as the services file and the IDE updated it when I renamed. I just moved it to its proper home though. Thanks!